tests/main_tests.py

```diff
@@ -1,59 +1,235 @@
 import unittest
-from app.main_amity import Room,Person
+#from app.classes.amity import Amity
+from app.classes.person import Person
+from app.classes.room import Room
+#from app.classes.main_amity import Room,Person
+from app.classes.database import Database
 
```

Comment:
Consider splitting up your test files into tests for Amity functions like adding persons and adding rooms, reallocate rooms and print allocation in one test file and database tests in another module for a total of two test modules.